src/Store/MultiTableStore.php

```diff
@@ -39,7 +40,7 @@ public function __construct(
     /**
      * @param class-string<AggregateRoot> $aggregate
      *
-     * @return AggregateChanged[]
+     * @return array<AggregateChanged>
```

Comment:
```suggestion
     * @return list<AggregateChanged>
```

Reply:
cant be done unless using `array_values` after `array_merge`. Seems not feasible for this